src/main/java/org/opennms/test/system/api/NewTestEnvironment.java

```diff
@@ -956,6 +956,7 @@ private void waitForSentinel() throws Exception {
         LOG.info("************************************************************");
         LOG.info("Waiting for Sentinel @ {} to start.", sshAddr);
         LOG.info("************************************************************");
+        await().atMost(5, MINUTES).pollInterval(5, SECONDS).until(() -> canSentinelConnectToOpennms(sshAddr));
```

Comment:
Sometimes Sentinel will be set up to send data directly to Elasticsearch and such, won't it?  Will there be cases where you'll do Sentinel tests without connecting it up to OpenNMS?

Reply:
We should then change the `sentinel:ping` command to ignore the rest check. At the moment it is assumed, that a connection is required, e.g. for later retrievement of configuration files. 

But there might be a case, where this is not required. 
We can skip this change, or revert it as soon as we know more about it.